synapse/tools/axon2to3.py

```diff
@@ -0,0 +1,69 @@
+import os
+import sys
+import asyncio
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+
+import synapse.lib.cmd as s_cmd
+import synapse.lib.cell as s_cell
+import synapse.lib.output as s_output
+import synapse.lib.msgpack as s_msgpack
+import synapse.lib.lmdbslab as s_lmdbslab
+
+async def migrate_history(dirn, outp=s_output.stdout):
```

Comment:
we should discuss moving this to being an inline storage update. I think that's the expected behavior with the other services. :+1: